cmd/up.go

```diff
@@ -158,41 +183,141 @@ func (cmd *UpCmd) Run(ctx context.Context, devPodConfig *config.Config, client c
 		log.Infof("Run 'ssh %s.devpod' to ssh into the devcontainer", client.Workspace())
 	}
 
+	dotfilesRepo := devPodConfig.ContextOption(config.ContextOptionDotfilesURL)
```

Comment:
Can you put this complete code in a separate function and then call that here?

Reply:
Done :+1: